doc/devel/dependencies.rst

```diff
@@ -47,6 +47,7 @@ and the capabilities they provide.
 * pycairo_ (>= 1.11.0) or cairocffi_ (>= 0.8): for the GTK and/or cairo-based
   backends.
 * Tornado_ (>=5): for the WebAgg backend.
+* OS X/macOS (>=10.12): for the macosx backend.
```

Comment:
```suggestion
* macOS (>=10.12): for the macosx backend.
```

It looks like with 10.12 they changed to just macOS, so we should probably just drop the "OS X" labels?
https://en.wikipedia.org/wiki/MacOS

Note that I just mean the documentation titles/labels here, not the backend name or anything more invasive.